### PR TITLE
Rename generic_work_solr_document to presenter

### LINKED ISSUE #2196: Rename generic_work_solr_document to work_solr_document

### PR DESCRIPTION
**Pull request by jcoyne**
@projecthydra/sufia-code-reviewers

Fixes #2196

### CODE REVIEW

File: app/views/sufia/homepage/_sortable_featured.html.erb [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
   <div class="dd-handle dd3-handle"><%= t :drag %></div>
   <div class="dd3-content panel panel-default">
     <%= f.hidden_field :id %>
-    <%= f.hidden_field :order, data: {property: "order"} %>
+    <%= f.hidden_field :order, data: { property: "order" } %>
     <div class="main row">
       <div class="col-sm-3">
-        <%= link_to curation_concerns_generic_work_path(featured) do %>
-          <%= render_thumbnail_tag featured, {width: 90} %>
+        <%= link_to curation_concerns_generic_work_path(presenter) do %>
```
Comment (mjgiarlo) [inline app/views/sufia/homepage/_sortable_featured.html.erb]: This is going to conflict with #2198. Do you have a preference which PR is merged first?

Comment (jcoyne) [inline app/views/sufia/homepage/_sortable_featured.html.erb]: nope

Comment (mjgiarlo) [inline app/views/sufia/homepage/_sortable_featured.html.erb]: K.

**Comment by mjgiarlo** [comment]
OK, rebase and re-push.

**Pull request merged**